src/watcher.ts

```diff
@@ -342,6 +420,9 @@ export class Watcher {
       case 'aborted': {
         return;
       }
+      case 'initial': {
```

Comment:
why is it unexpected to abort before starting the first run?

Reply:
Because we immediately transition to `running`. But I don't want the initial state to be `running` because in every other situation calling `startRun` while the state is already `running` indicates something bad.